src/Numberable/Modulo.php

```diff
@@ -0,0 +1,22 @@
+<?php declare(strict_types=1);
+
+namespace GW\Value\Numberable;
+
+use GW\Value\Numberable;
+
+final class Modulo implements Numberable
+{
+    private Numberable $dividend;
+    private Numberable $divisor;
+
+    public function __construct(Numberable $dividend, Numberable $divisor)
+    {
+        $this->dividend = $dividend;
+        $this->divisor = $divisor;
+    }
+
+    public function toNumber(): int
+    {
+        return $this->dividend->toNumber() % $this->divisor->toNumber();
```

Comment:
What if `divisor->toNumber()` will be `0`?

Reply:
It will be a `DivisionByZeroError`.
TBH I don't remember why I added explicit throwing of this error in other classes. Is it better or should I remove it?